views/email/layout/empty/htmlWithHead.cfm

```diff
@@ -0,0 +1,14 @@
+<cfparam name="args.body" type="string" />
+<cfparam name="args.head" type="string" />
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Maybe the transitional doctype would be better?

https://responsivehtmlemail.com/html-email-doctype/